### DISCUSSION TITLE
React Admin Dashboard Template Candidates

### DISCUSSION THREAD

**Discussion by jihoon-seo** [cloud-barista/cb-tumblebug#474] [General]
CC: @hermitkim1 @seokho-son 

- https://usecode.pw/free-react-dashboard/
- Argon Dashboard React
  - Details: https://www.creative-tim.com/product/argon-dashboard-react
  - Live demo: https://demos.creative-tim.com/argon-dashboard-react
- Black Dashboard React
  - Details: https://www.creative-tim.com/product/black-dashboard-react
  - Live demo: https://demos.creative-tim.com/black-dashboard-react/#/admin/dashboard
- Material Dashboard React
  - Live demo: https://material-ui.com/getting-started/templates/dashboard/
  - Source code: https://github.com/mui-org/material-ui/tree/master/docs/src/pages/getting-started/templates/dashboard

**Comment by jihoon-seo** [comment]
https://usecode.pw/free-react-dashboard/ 에 있는 AntD Admin 을 시도해 봤는데
직관적이지 않고 어렵습니다.
다른 것을 시도해 보겠습니다.

**Reply by seokho-son** [reply]
감사합니다.. :)

**Comment by jihoon-seo** [comment]
[CoreUI Free React Admin Template v3](https://github.com/coreui/coreui-free-react-admin-template) 으로 해 보고 있습니다~~

**Reply by seokho-son** [reply]
멋져요!
